readme.md

```diff
@@ -1 +1,66 @@
-http://todomvc.com but for back end.
+# Goal
+
+* Create various implementations of a minimally simple addressbook service
+
+# Why?
+
+* To ease comparison between different languages/frameworks/approaches
+* To demonstrate how to use the various languages/frameworks/approaches
+
+## The Original Idea
+
+http://todomvc.com but using server-side technologies.
+
+# Implementation Guidelines
+
+## General
+
+* Go for clarity and simplicity which should ease comparison and improve the pedagogical win.
+* Err on the side of excessive commenting.
+* Implementations MUST include persistence.
+* Implementations SHOULD behave as described in the API doc below, whether it provides an actual API service or merely outputs HTML directly.
```

Comment:
I think I'd like to de-emphasize anything about a REST API. The app should still function even if it doesn't expose an API.